CHANGELOG.md

```diff
@@ -12,16 +16,30 @@
 * Photon field classes replaced the photon field enumerator,
   consequently, the new photon fields implementation follows the same logic as
   of the other modules making it is easier to introduce custom ones
+* New class-based interface for turbulent fields introduced
+* New turbulence modules implemented:
+  - GridTurbulence (with the bendover scale) which should in general be used
+    instead of initTurbulence (before, it was implmeneted as
+    initTurbulenceWithBendover);
+  - SimpleGridTurbulence which provides the exact field as initTurbulence;
+  - HelicalGridTurbulence which provides the exact field as initHelicalTurbulence
+  - PlaneWaveTurbulence a new algorthm based on Giacalone & Jokipii, 1999 and
+    Tautz & Dosch, 2013.
 
 ### Interface change:
 
 * ParticleCollector::getAll() -> ParticleCollector::getContainer()
 * Photon fields are no longer items of the PhotonField enumerator but independent
   classes that shares the same interface, so instead of `CMB` one should use `CMB()`,
```

Comment:
@adundovi can you please fix that typo here in the same run, too: shares -- share, since photon fields are plural